pkg/token/verifier.go

```diff
@@ -67,12 +67,16 @@ func (v *verifier) Verify(rawIDToken string) (*IDToken, error) {
 	if err := json.Unmarshal(payload, t); err != nil {
 		return nil, fmt.Errorf("failed to unmarshal claims: %v", err)
 	}
-	if t.Issuer != v.issuer {
-		return nil, fmt.Errorf("id token issued by a different provider, expected %q got %q", v.issuer, t.Issuer)
-	}
-	if t.Audience != v.clientID {
-		return nil, fmt.Errorf("expected audience %q got %q", v.clientID, t.Audience)
-	}
+	// Since we will keep old and new authentication API work at then same time, we don't check the issuer and clientID
+	// temporarily (new authentication will have different issuer and clientID).
+	// After new api is working as expected, we will upgrade the Verifier.
+
+	//if t.Issuer != v.issuer {
+	//	return nil, fmt.Errorf("id token issued by a different provider, expected %q got %q", v.issuer, t.Issuer)
+	//}
+	//if t.Audience != v.clientID {
+	//	return nil, fmt.Errorf("expected audience %q got %q", v.clientID, t.Audience)
+	//}
```

Comment:
@cre8ivejp We will skip `issuer` and `clientID` check here, because old authentication use dex as issuer, new authentication will use google as issuer. 

So if we want to make old and new authentication api work at the same time, we should skip this check temporarily.